### ISSUE TITLE
issues uploading multiple files

### ISSUE THREAD

**Issue by tachim** [codalab/codalab-worksheets#13]
```
tudor@tudor-258:~/prediction$ cl up dataset deptrees/train deptrees/dev                                                 
Traceback (most recent call last):                                                                                      
  File "/home/tudor/codalab-cli/codalab/bin/../../codalab/bin/cl.py", line 67, in <module>                              
    run_cli()                                                                                                           
  File "/home/tudor/codalab-cli/codalab/bin/../../codalab/bin/cl.py", line 58, in run_cli                               
    cli.do_command(sys.argv[1:])                                                                                        
  File "/home/tudor/codalab-cli/codalab/lib/bundle_cli.py", line 329, in do_command                                     
    command_fn(remaining_args, parser)                                                                                  
  File "/home/tudor/codalab-cli/codalab/lib/bundle_cli.py", line 458, in do_upload_command                              
    print client.upload_bundle(args.path, {'bundle_type': args.bundle_type, 'metadata': metadata}, worksheet_uuid, args.
follow_symlinks)                                                                                                        
  File "/home/tudor/codalab-cli/codalab/client/remote_bundle_client.py", line 157, in upload_bundle                     
    if path_util.path_is_url(path):                                                                                     
  File "/home/tudor/codalab-cli/codalab/lib/path_util.py", line 421, in path_is_url                                     
    if path.startswith(prefix + '://'):                                                                                 
AttributeError: 'list' object has no attribute 'startswith'                                                             
```

**Comment by tachim** [comment]
The intention is to upload both files into the same bundle.

**Comment by percyliang** [comment]
Fixed by #14 - just pushed to master.
